ios/Sources/Component/Tag/Tag.swift

```diff
@@ -68,6 +71,8 @@ private extension TagType {
             return AssetColor.Tag.medium.color
         case .youtube:
             return AssetColor.Tag.video.color
+        case .other:
+            return AssetColor.Tag.other.color
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)
https://github.com/DroidKaigi/conference-app-2021/blob/4f1f481f51646a780f2aec9613555417b1abb2c0/uicomponent-compose/core/src/main/java/io/github/droidkaigi/feeder/core/theme/Color.kt#L12
Using the value from here, but it doesn't have enough contrast in dark mode (theme).
Should I ask the designer for a color for dark mode (theme)?

Reply:
I see. I missed that color 😢 
Yes, please 🙏🏼  I think we need that color!